src/app/Agent/AgentLiveProbes.tsx

```diff
@@ -0,0 +1,266 @@
+    [templates]
+  );
+
+  const handleModalToggle = () => {
+    addSubscription(
+      context.api
+        .removeProbes()
+        .pipe(first())
+        .subscribe(() => {
+          refreshTemplates();
+        })
+    );
+  };
+
+  React.useEffect(() => {
+    addSubscription(
+      context.notificationChannel
+        .messages(NotificationCategory.ProbeTemplateApplied)
+        .subscribe((v) => refreshTemplates())
+    );
+  }, [addSubscription, context, context.notificationChannel, setTemplates]);
+
+  const authRetry = React.useCallback(() => {
+    context.target.setAuthRetry();
+  }, [context.target, context.target.setAuthRetry]);
+
+  if (errorMessage != '') {
+    return (
+      <ErrorView
+        title={'Error retrieving events'}
+        message={
+          'This is commonly caused by the agent not being loaded/active, check that the target was started with the agent (-javaagent:/path/to/agent.jar) ' +
+          errorMessage
+        }
+        retry={isAuthFail(errorMessage) ? authRetry : undefined}
+      />
+    );
+  } else if (isLoading) {
+    return <LoadingView />;
+  } else {
+    return (
+      <>
+        <Card>
+          <CardHeader>
+            <CardHeaderMain>
+              <Text component={TextVariants.h4}>About the JMC Agent</Text>
+            </CardHeaderMain>
+          </CardHeader>
+          <CardBody>
+            The JMC Agent allows users to dynamically inject custom JFR events into running JVMs. In order to make use
+            of the JMC Agent, the agent jar must be present in the same container as the target, and the target must be
+            started with the agent (-javaagent:/path/to/agent.jar). Once these pre-requisites are met the user can
+            upload probe templates to Cryostat and insert/remove them from targets, as well as view currently active
+            probes.
+          </CardBody>
+        </Card>
+        <Toolbar id="event-templates-toolbar">
+          <ToolbarContent>
+            <ToolbarGroup variant="filter-group">
+              <ToolbarItem>
+                <TextInput
+                  name="templateFilter"
+                  id="templateFilter"
+                  type="search"
+                  placeholder="Filter..."
+                  aria-label="Probe template filter"
+                  onChange={setFilterText}
+                />
+              </ToolbarItem>
+            </ToolbarGroup>
+            <ToolbarGroup variant="icon-button-group">
+              <ToolbarItem>
+                <Button key="create" variant="primary" onClick={handleModalToggle}>
+                  Remove Probes
+                </Button>
+              </ToolbarItem>
+            </ToolbarGroup>
+          </ToolbarContent>
+        </Toolbar>
+        <Table aria-label="Active Events" variant={TableVariant.compact} cells={tableColumns} rows={displayTemplates}>
```

Comment:
Table should have specify `onSort={handleSort}` to set the current `sortBy`. Right now, toggling sort does not work.